environment.yml

```diff
@@ -2,6 +2,7 @@ name: symbiflow_arch_def_base
 channels:
   - defaults
   - symbiflow
+  - conda-forge
```

Comment:
Just to mention this, I recall there was an effort to avoid using the conda-forge channel as it led to possible conflicts among packages.

I think there is no other way though to have the capnproto-java package installed though, so I guess this is fine.

Reply:
If I add `capnproto` to the symbiflow channel we can avoid conda-forge.  The `capnproto` in `conda-forge` is pretty out of date anyways, so this is probably worthwhile anyways :/

Reply:
FYI: https://github.com/SymbiFlow/conda-packages/pull/137